app.rb

```diff
@@ -0,0 +1,49 @@
+require 'sinatra'
+require 'sinatra/reloader'
+require 'active_record'
+
+# Load the file to connect to the DB
+require_relative 'db/connection'
+
+# Load models
+require_relative 'models/pokemon'
+
+get '/' do
+  @pokemonsvariable=Pokemon.all
+  erb :"index"
+end
+
+get '/new' do
```

Comment:
RESTful convention is to say `get "/pokemons/new"`

Imagine if you had another model. How would you differentiate between the new forms for those two models?